signer/src/bitcoin/validation.rs

```diff
@@ -834,7 +835,7 @@ impl DepositRequestReport {
         match self.lock_time {
             LockTime::Blocks(height) => {
                 let max_age = height.value().saturating_sub(DEPOSIT_LOCKTIME_BLOCK_BUFFER) as u64;
-                if deposit_age >= max_age {
+                if deposit_age >= max_age.into() {
```

Comment:
It feels a little like we should be working with integers and not heights here as we're dealing with "age". Maybe do the `.into()` on the `.saturating_sub()` above?